settings.gradle

```diff
@@ -50,16 +50,17 @@ includeWithFlags ':docs-client'
 includeWithFlags ':site'
 
 // Examples
-includeWithFlags ':examples:annotated-http-service', 'java'
-includeWithFlags ':examples:dropwizard-example',      'java'
+includeWithFlags ':examples:annotated-http-service',        'java'
+includeWithFlags ':examples:dropwizard-example',            'java'
 project(':examples:dropwizard-example').projectDir = file('examples/dropwizard')
-includeWithFlags ':examples:grpc-example',           'java'
+includeWithFlags ':examples:grpc-example',                  'java'
 project(':examples:grpc-example').projectDir = file('examples/grpc')
-includeWithFlags ':examples:grpc-reactor',           'java', 'reactor-grpc'
-includeWithFlags ':examples:proxy-server',           'java'
-includeWithFlags ':examples:saml-service-provider',  'java'
-includeWithFlags ':examples:server-sent-events',     'java'
-includeWithFlags ':examples:spring-boot-minimal',    'java'
-includeWithFlags ':examples:spring-boot-tomcat',     'java'
-includeWithFlags ':examples:spring-boot-webflux',    'java'
-includeWithFlags ':examples:static-files',           'java'
+includeWithFlags ':examples:grpc-reactor',                  'java', 'reactor-grpc'
+includeWithFlags ':examples:proxy-server',                  'java'
+includeWithFlags ':examples:saml-service-provider',         'java'
+includeWithFlags ':examples:server-sent-events',            'java'
+includeWithFlags ':examples:spring-boot-minimal',           'java'
+includeWithFlags ':examples:spring-boot-minimal-kotlin',    'java'
```

Comment:
It is trivial but the `java` flag looks a little weird. If more Kotlin examples and DSL is added, then don't we need `kotlin` flag?

Reply:
Do I need to clear the `java` flag and put `kotlin` flag?

To that end, I need to modify [`/gradle/scripts`](https://github.com/line/armeria/tree/master/gradle/scripts). But isn't this file related to [`line/gradle-scripts`](https://github.com/line/gradle-scripts)? 

Or is there another way?

Reply:
I’m not 100% sure we need to add `kotlin` flag. Does it seems overkill? @trustin @minwoox

Reply:
I think it's fine because this is just an example and the build.gradle.kts are copied line by line by our script.
But I guess we need it someday because we are doing [this](https://github.com/line/gradle-scripts/blob/master/lib/java.gradle#L14). We might add kotlin.gradle in that directory. 😄

Reply:
> I think it's fine because this is just an example and the build.gradle.kts are copied line by line by our script.

That sounds good to me. Let leave it as it is. :-)